models/edu/stanford/nlp/models/sutime/english.sutime.txt

```diff
@@ -14,8 +14,8 @@
 
   $INT_TIMES = ( $INT /times/ | once | twice | thrice );
   $REL_MOD = ( /the/? /next|following|last|previous/ | /this/ /coming|past/? | /the/ /coming|past/ );
-  $FREQ_MOD = ( /each|per/ | /every|per/ $NUM_ORD | /every|per/ /other|alternate|alternating/? | /alternate|alternating/ );
-  $EARLY_LATE_MOD = ( /late|early|mid-?/ | /the/? /beginning|start|dawn|middle|end/ /of/ );
+  $FREQ_MOD = ( /each/ | /per/ | /every/ $NUM_ORD | /every/ /other|alternate|alternating/? | /alternate|alternating/ );
```

Comment:
Now does this this pattern support stuff like "each other month"?

Reply:
Utterances like "each other month" and "per alternating year" are now supported.